app/models/measure_condition.rb

```diff
@@ -3,7 +3,11 @@
 class MeasureCondition
   include ApiEntity
 
+  WEIGHT_UNITS = %w[DTN DAP DHS GFI GRM GRT KGM KMA RET TNE].freeze
```

Comment:
This feels like something we should add to the measure condition response

Reply:
Probably yes - I'll create a separate ticket for that, once we have it in the backend we can switch to using it from the frontend